<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Mekan Myradov</title>
  
  <meta name="author" content="Mekan Myradov">
  <meta name="viewport" content="width=device-width, initial-scale=1">    
  
  <link rel="stylesheet" type="text/css" href="style.css">
  <link rel="icon" type="image/png" href="images/IDSIA_logo.png">
</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px"><td style="padding:0px">
      <!-- Intro -->
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr style="padding:0px">
          <td style="padding:2.5%;width:63%;vertical-align:middle">
            <p style="text-align:center">
              <name>Mekan Myradov</name>
            </p>
            <p>
              I'm currently an MSc student at <a href="https://www.sabanciuniv.edu/en/"><span style="white-space: nowrap;">Sabanci University</span></a>, 
              affiliated with the <a href="https://www.vpalab.com/"><span style="white-space: nowrap;">VPA Lab</span></a> under the guidance of 
              <a href="http://people.sabanciuniv.edu/otastan/"><span style="white-space: nowrap;">Öznur Taştan</span></a> and <a href="https://antalya.edu.tr/en/academic-staff/college-of-engineering/hilal-kazan#"><span style="white-space: nowrap;">Hilal Kazan</span></a>. 
              My research focuses on machine learning applications in biology, specifically single-cell RNA sequencing data integration.
            </p>
            
            <p style="text-align:center">
              <a href="mailto:mekan.myradov@sabanciuniv.edu">Email</a> &nbsp/&nbsp
              <a href="./cv/00_Mekan_Myradov_CV.pdf">CV</a> &nbsp/&nbsp
              <a href="https://github.com/MekanMyradov">GitHub</a> &nbsp/&nbsp
              <a href="https://scholar.google.com/citations?user=3xLYcY8AAAAJ&hl=en">Google Scholar</a>
            </p>
          </td>
          <td style="padding:2.5%;width:40%;max-width:40%">
            <a href="./photos/mekan.jpeg"><img style="width:100%;max-width:100%" alt="profile photo" src="./photos/mekan.jpeg" class="hoverZoomLink"></a>
          </td>
        </tr>
      </tbody></table>

      <!-- Publications title -->
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
          <td style="padding:20px;width:100%;vertical-align:middle">
            <heading>Publications and Projects</heading>
          </td>
        </tr>
      </tbody></table>
        
      <!-- List of publications and projects -->
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        
        <!-- SCITUNA: single-cell data integration tool using network alignment -->
        <tr>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-025-06087-3">
              <papertitle>SCITUNA: single-cell data integration tool using network alignment</papertitle>
            </a>
            <br>
            Aissa Houdjedj, Yacine Marouf, <strong>Mekan Myradov</strong>, Onur Dogan, Burak Onur Erten, Oznur Tastan, Cesim Erten and Hilal Kazan
            <br>
            <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-025-06087-3">pdf</a> /
            <a href="https://github.com/abu-compbio/SCITUNA">code</a> /
            <a href="./refs/scituna.bib">bib</a>
            <br>
            <p>
              We introduce a novel method for batch effect correction named SCITUNA, a Single-Cell data Integration Tool Using Network Alignment.
              The method aligns batches iteratively, progressively integrating each batch into a unified representation to correct for batch effects.
              We perform evaluations on 39 individual batches from four real datasets and a simulated dataset, which include both scRNA-seq and
              scATAC-seq datasets, spanning multiple organisms and tissues. A thorough comparison of existing batch correction methods using 13
              metrics reveals that SCITUNA outperforms current approaches and is successful at preserving biological signals present in the original data.
            </p>
          </td>
        </tr>
        
        <!-- ADRMX: Additive Disentanglement of Domain Features with Remix Loss -->
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/ADRMX_2.png" height="180" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://arxiv.org/abs/2308.06624">
              <papertitle>ADRMX: Additive Disentanglement of Domain Features with Remix Loss</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>,
            <a href="https://sites.google.com/view/erchan-aptoula/">Erchan Aptoula</a>,
            <a href="http://myweb.sabanciuniv.edu/hozkan/">Huseyin Ozkan</a>
            <br>
              <em> arXiv:2308.06624, 2023 </em>
            <br>
            <a href="https://arxiv.org/pdf/2308.06624.pdf">pdf</a> /
            <a href="https://github.com/berkerdemirel/adrmx">code</a> /
            <a href="./refs/adrmx.bib">bib</a>
            <br>
            <p>
              We propose an additive disentanglement of domain specific and domain invariant features for the domain generalization problem. Unlike prior work, 
              we demonstrate the potential benefits of utilizing domain specific features along with domain invariant ones. Moreover, we introduce a new data 
              augmentation technique to enhance the generalization capacity of the architecture, where samples from different domains are mixed 
              within the latent space.
            </p>
          </td>
        </tr>
        -->

        <!-- DECOMPL: Decompositional Learning with Attention Pooling for Group Activity Recognition from a Single Volleyball Image -->
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/DECOMPL.png" height="120" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://arxiv.org/abs/2303.06439">
              <papertitle>DECOMPL: Decompositional Learning with Attention Pooling for Group Activity Recognition from a Single Volleyball Image</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>,
            <a href="http://myweb.sabanciuniv.edu/hozkan/">Huseyin Ozkan</a>,
            <br>
              <em> arXiv:2303.06439, 2023 </em>
            <br>
            <a href="https://arxiv.org/pdf/2303.06439.pdf">pdf</a> /
            <a href="https://github.com/berkerdemirel/decompl">code</a> /
            <a href="./refs/decompl.bib">bib</a>
            <br>
            <p>
              We show that Group Activity Recognition Problem can be formulated using Attention Pooling mechanism and can perform on par with
              the other state-of-the-art methods even with a single RGB frame. Moreover, we manually reannotated the flawed instances in the Volleyball Dataset,
              which is one of the widely used datasets in Group Activity Recognition.
            </p>
          </td>
        </tr>
        -->
        
        <!-- Vertex Ordering Algorithm for Graph Coloring Problem -->
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/graph_coloring.png" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://ieeexplore.ieee.org/abstract/document/9302296">
              <papertitle>Vertex Ordering Algorithms for Graph Coloring Problem</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>,
            <a href="https://www.linkedin.com/in/ardaasik/">Arda Asik</a>,
            <a href="https://www.linkedin.com/in/bdemir97/">Bugra Demir</a>,
            <a href="https://fens.sabanciuniv.edu/en/faculty-members/detail/2380">Kamer Kaya</a>,
            <a href="https://www.linkedin.com/in/barisbatuhan/?originalSubdomain=tr">Baris Batuhan Topal</a>
            <br>
              <em> IEEE Signal Processing and Communications Applications (SIU), 2020 </em>
            <br>
            <a href="https://arxiv.org/pdf/2008.11454.pdf">pdf</a> /
            <a href="https://github.com/barisbatuhan/Graph_Coloring_with_RL">code</a> /
            <a href="./refs/voagcp.bib">bib</a>
            <br>
            <p>
              We developed a ranking algorithm that uses metrics of degree 1/2/3, closeness centrality,
              clustering coefficient, and page rank to rank the nodes in a graph. Then, we applied greedy coloring 
              to the graph using the ranking as a guide, which resulted in significantly better colorings.
              Furthermore, we tried to extend this idea using a model-free policy based reinforcement learning algorithm while 
              parallelizing C++ backend using OpenMP.
            </p>
          </td>
        </tr>
        -->

        <tr>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/MekanMyradov/anova-linear-regression-stats-toolbox">
              <papertitle>ANOVA and Linear Regression Toolbox</papertitle>
            </a>
            <br>
            <strong>Mekan Myradov</strong>
            <br>
            <a href="https://github.com/MekanMyradov/anova-linear-regression-stats-toolbox">pdf / code</a>
            <br>
            <p>
              I implemented a comprehensive Python toolbox for statistical analysis implementing one-way ANOVA, two-way ANOVA,
              and multiple linear regression with various hypothesis testing and confidence interval methods.
            </p>
          </td>
        </tr>

        <tr>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/MekanMyradov/Simulations">
              <papertitle>Synthetic Data Generation with Gaussian Mixture Models</papertitle>
            </a>
            <br>
            <strong>Mekan Myradov</strong>
            <br>
            <a href="https://www.nature.com/articles/nbt.4091">pdf</a> /
            <a href="https://github.com/MekanMyradov/Simulations">code</a>
            <br>
            <p>
              I implemented a synthetic data generation algorithm in Python using Gaussian Mixture Models.
            </p>
          </td>
        </tr>
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/mnist.png" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/berkerdemirel/UMAP">
              <papertitle>Implementation of UMAP in Python (Course Project)</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>
            <br>
            <a href="https://arxiv.org/pdf/1802.03426.pdf">pdf</a> /
            <a href="https://github.com/berkerdemirel/UMAP">code</a>
            <br>
            <p>
              I implemented Uniform Manifold Approximation and Projection (UMAP) algorithm in Python from scratch 
              and performed experiments on MNIST and Load Digits datasets.
            </p>
          </td>
        </tr>
        -->
        
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/dcgan.png" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/berkerdemirel/Deep-Convolutional-Generative-Adversarial-Networks">
              <papertitle>Implementation of DCGAN using PyTorch (Course Project)</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>
            <br>
            <a href="https://arxiv.org/pdf/1511.06434.pdf">pdf</a> /
            <a href="https://github.com/berkerdemirel/Deep-Convolutional-Generative-Adversarial-Networks">code</a>
            <br>
            <p>
              I implemented convolutional Generator-Discriminator based GAN using PyTorch, and performed experiments on MNIST, CIFAR-10 and Celeb-A datasets.
            </p>
          </td>
        </tr> 
        -->

        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/EM.png" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/berkerdemirel/Expectation-Maximization-Algorithm">
              <papertitle>Implementation of Expectation-Maximization Algorithm using Matlab (Course Project)</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>
            <br>
            <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=543975">pdf</a> /
            <a href="https://github.com/berkerdemirel/Expectation-Maximization-Algorithm">code</a>
            <br>
            <p>
              I implemented Expectation-Maximization algorithm for Gaussian Mixture Model using Matlab.
            </p>
          </td>
        </tr>
        -->
        
        <!--
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="./photos/DOBFS.png" width="180">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <a href="https://github.com/berkerdemirel/Parallel-Breadth-First-Search-OpenMP-and-CUDA">
              <papertitle>Implementation of Direction-Optimized Breadth First Search using OpenMP and CUDA (Course Project)</papertitle>
            </a>
            <br>
            <strong>Berker Demirel</strong>,
            <a href="https://www.linkedin.com/in/negesarac/?originalSubdomain=tr">Naci Ege Sarac</a>
            <br>
            <a href="https://scottbeamer.net/pubs/beamer-sc2012.pdf">pdf</a> /
            <a href="https://github.com/berkerdemirel/Parallel-Breadth-First-Search-OpenMP-and-CUDA">code</a>
            <br>
            <p>
              We implemented Direction-Optimized BFS using OpenMP and CUDA.
            </p>
          </td>
        </tr>
        -->

  <!-- Credits -->
  <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr>
      <td style="padding:0px">
        <br>
        <p style="text-align:right;font-size:small;">
          Website template credits to <a href="https://jonbarron.info/">Jon Barron</a>.
        </p>
      </td>
    </tr>
  </tbody></table>
</body>

</html>